### ISSUE TITLE
False positive with JSDoc imports

### ISSUE THREAD

**Issue by regseb** [denisoed/dead-code-checker#27]
When the JSDoc import of a type is used, dead-code-checker reports that the type isn't used.

- `foo.js`

  ```javascript
  export class Foo {};
  ```

- `index.js`

  ```javascript
  /**
   * @import { Foo } from "./foo.js"
   */

  /**
   * @type {Foo}
   */
  const foo;
  ```

---

`npx dead-code-checker -f .`

> ...
> 📁 index.js (2 items):
> &nbsp;&nbsp;&nbsp;&nbsp;📥 import Foo (imported but not used)
> ...
